Resources/views/CRUD/base_edit.html.twig

```diff
@@ -12,7 +12,7 @@ file that was distributed with this source code.
 {% extends base_template %}
 
 {% block title %}
-    {% if admin.id(object) is not null %}
+    {% if objectId|default(admin.id(object)) is not null %}
```

Comment:
Add it everywhere please. Act as if a five year old was doing the next major cleanup.